configs/components/oasis3mct/oasis3mct.env.yaml

```diff
@@ -16,3 +29,19 @@ add_compiletime_environment_changes:
                         intel-oneapi:
                             add_export_vars:
                                 OASIS_FFLAGS: '"-march=core-avx2"'
+
+choose_oasis3mct.with_yac:
+        True:
+                yac_libs: " -lyac_utils -lyac_core -lyaxt -lyaxt_c "
+                yac_inc_path: "-I$OIFS_OASIS_BASE/lib/include/yac -I$OIFS_OASIS_BASE/lib/yaxt/include"
+                yac_lib_path: "-L$OIFS_OASIS_BASE/lib/yac/lib -L$OIFS_OASIS_BASE/lib/yaxt/lib"
+        False:
+                yac_libs: ""
+                yac_inc_path: ""
+                yac_lib_path: ""
+
+
+add_runtime_environment_changes:
+        add_export_vars:
+                - 'LD_LIBRARY_PATH=${model_dir}/oasis/lib/yaxt/lib/:${model_dir}/oasis/lib/yac/lib/:$LD_LIBRARY_PATH'
```

Comment:
Shouldn't this go into the choose block above?

Reply:
It does no harm here, but it makes more sense up there. I will try if that works.